code/utils/graphing/plot_neural_behavior.py

```diff
@@ -0,0 +1,38 @@
+"""
+Function to plot the linear regression picture of behavial loss aversion
+and neural loss aversion(-betagain-betaloss) at a single coordinate of standard brain
+"""
```

Comment:
need to `import numpy as np`  & `from scipy import stats` & `import matplotlib.pyplot as plt`